apps/pie-docs/src/components/assistive-text/overview/overview.md

```diff
@@ -0,0 +1,214 @@
+- Use sentence case.
+
+### Success
+
+- Should describe the success as clearly and succinctly as possible.
+- Use sentence case.
+
+---
+
+## Overflow
+
+### Multiple words overflow
+
+When a group of words extends beyond the available horizontal space, the text automatically wraps onto a new line, with no maximum height restriction.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/overflow-1.svg",
+      width: "256px",
+      alt: "Assistive text component with overflow displayed."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/overflow-2.svg",
+      width: "256px",
+      alt: "Assistive text component with overflow displayed."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Behaviours
+
+If a validation variant of the assistive text is required for error or success, it appears once the field has been validated; and will replace the default assistive text if one was present.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/behaviours.svg",
+    alt: "Assistive text component showing its behaviours in various states.",
+    width: "570px"
+} %}
+
+---
+
+## Examples
+
+Outlines the atomic level interactive elements for the component.
+
+### LTR examples
+
+Here are some examples of assistive text in left-to-right context:
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/examples-LTR-checkbox.svg",
+      width: "135px",
+      alt: "Example of a left-to-right checkbox field with assistive text."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/examples-LTR-input.svg",
+      width: "256px",
+      alt: "Example of a left-to-right input field with assistive text."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### RTL examples
+
+Here are some examples of assistive text in right-to-left context:
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/examples-RTL-checkbox.svg",
+      width: "135px",
+      alt: "Example of a right-to-left checkbox field with assistive text."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/assistive-text/examples-RTL-input.svg",
+      width: "256px",
+      alt: "Example of a right-to-left input field with assistive text."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Resources
+
+{% notification {
+  type: "warning",
+  message: "We’re currently working on updating our Assistive Text documentation, please see the resources below."
+} %}
+
+{% resourceTable {
+    componentName: 'Assistive Text'
```

Comment:
With the Snacks link, is `bubble` the right component to link to?

Reply:
I can't find assistive-text component in Snacks, maybe this can be used instead? https://snacks.takeaway.com/portal/components/error-message/

Reply:
This link from Pie Docs doesn’t have an equivalent on Snacks.
Input components already have that AssistiveMessage baked in.

So I will remove this Snacks link to assistive text at all.

Reply:
That's great. I wonder if the status of `Stable` is still accurate if there is no direct Snacks equivalent?

Reply:
you are right, would be better to add `N/A` status like for Vue [Fozzie]